1-js/05-data-types/11-date/article.md

```diff
@@ -54,19 +54,19 @@ To create a new `Date` object call `new Date()` with one of the following argume
     // Wed Jan 25 2017 16:00:00 GMT-0800 (Pacific Standard Time)
     ```
 
-`new Date(year, month, date, hours, minutes, seconds, ms)`
+`new Date(year, month, day, hours, minutes, seconds, ms)`
```

Comment:
would date be more clear? Since new Date().getDay() returns the number of the weekday? 🤔